securepy/restrictor.py

```diff
@@ -1,16 +1,24 @@
 import traceback
 import typing as t
 
+from securepy.stdio import IOCage
 from securepy.security import RESTRICTED_GLOBALS, SAFE_GLOBALS
```

Comment:
Proper import order is required by `flake8-import-order` checks, we follow pycharms import ordering and which requires groups to have import statements in alphabetical order.
```suggestion
from securepy.security import RESTRICTED_GLOBALS, SAFE_GLOBALS
from securepy.stdio import IOCage
```